Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -28,6 +23,11 @@
 #import "CGImageInternal.h"
 #import "MobileCoreServices/UTType.h"
 
+#include <COMIncludes.h>
+#import <windows.storage.streams.h>
+#import "RawBuffer.h"
+#include <COMIncludes_End.h>
```

Comment:
COMIncludes.h cannot be imported

---
In reply to: [80762795](https://github.com/Microsoft/WinObjC/pull/1040#discussion_r80762795) [](ancestors = 80762795)